packages/calcite-components/src/components/action/action.tsx

```diff
@@ -272,8 +302,9 @@ export class Action extends LitElement implements InteractiveComponent {
           aria-controls={indicator ? indicatorId : null}
```

Comment:
Not sure if this is relevant here - but would this also need to be set in some way to reference the controlled "widget" (as MDN calls it) https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-expanded#buttons